app/models/bucket-model.coffee

```diff
@@ -0,0 +1,14 @@
+null
+
+# @ngInject
+global.cobudgetApp.factory 'BucketModel', (BaseModel) ->
+  class BucketModel extends BaseModel
+    @singular: 'bucket'
+    @plural: 'buckets'
+    @indices: ['groupId']
+
+    amountRemaining: ->
+      @target - @totalContributions
+
+    percentFunded: ->
+      @totalContributions / @target * 100
```

Comment:
i recommend we use a decimal lib like [decimal.js](https://www.npmjs.com/package/decimal.js) asap.

@robguthrie is it possible to custom parse the incoming json, so that the model uses decimals after being deserialized? and then the reverse for serializing.